concepts/05 Widgets/DropDownBox/10 Overview.md

```diff
@@ -319,17 +333,129 @@ If your data is an array of objects, specify:
                     key: "ID"
                 }),
                 isDropDownBoxOpened: false,
-                selectedCustomers: [customers[0].ID]
+                selectedCustomers: [customers[0].ID],
+                selectedValue: customers[0].ID
             }
         },
         methods: {
-            closeDropDownBox(e) {
+            changeDropDownBoxValue(e) {
+                this.selectedValue = this.selectedCustomers[0];
                 this.isDropDownBoxOpened = false;
             }
         }
     }
     </script>
 
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DropDownBox } from 'devextreme-react/drop-down-box';
+    import { DataGrid, Selection } from "devextreme-react/data-grid";
+    import ArrayStore from "devextreme/data/array_store";
+    
+    const customers = [
+        { ID: 1, companyName: "Premier Buy", city: "Dallas", phone: "(233)2123-11" },
```

Comment:
I suggest you to add one more item for simple scenario demonstration

Reply:
I don't object. It may look like this:

{ ID: 2, companyName: "ElectrixMax", city: "Naperville", phone: "(630)438-7800" },

Reply:
Thanks, I made it